docs/tutorials/todomvc-gaudi.md

```diff
@@ -0,0 +1,235 @@
+# How "todo" Gaudi
+
+## Intro
+
+You've probably all heard of the famous "TodoMVC" application. It was originally written by Addy Osmani of Google, not as a specific application but more as a project that provides a variety of implementations of a simple to-do list application. It is used to show and compare different JavaScript frameworks, technologies and even performance benchmarks. It's gotten so famous that it even got it's own [website](https://todomvc.com/) and it's own [application specification](https://github.com/tastejs/todomvc/blob/master/app-spec.md).
+
+The other day we were talking about writing a tutorial on creating an application that would showcase the power and ease of programming backend APIs in Gaudi. Since the whole point of a backend is to be used by some client or a frontend, we obviously need to create or find a frontend that would complete our demo project. After some brainstorming, we've all agreed that "TodoMVC" was the perfect fit because it was simple, well known, publicly [specified](https://github.com/tastejs/todomvc/blob/master/app-spec.md) and needed no additional introductions. Moreover, it already had a prepared [template](https://github.com/tastejs/todomvc-app-template) and [styles](https://github.com/tastejs/todomvc-app-css/) so we could focus on implementing application logic and stil have our aplication look nice. Win, win. :)
+
+Since "TodoMVC" primarily defines a GUI application and as such implemented mostly as a GUI-only applications, to communicate with our Gaudi backend we could either customize one of the existing examples or write a completely new GUI for our tutorial. Examples available at [todomvc.com](https://todomvc.com/) were pretty outdated and unmaintained so we've decided to go with our own GUI implemented with [React](htps://react.dev) and [Typescript](https://www.typescriptlang.org/).
+
```

Comment:
This whole introduction is rather long and probably not necessary, but I enjoyed reading it :) Should we keep it? How do you feel about it now that it's 2 weeks old? :P

Reply:
I like stories so I'm biased. :-) I've refactored it a bit so it's a little bit "drier" than before but still has an intro story.